lib/search_cop/grammar_parser.rb

```diff
@@ -12,9 +12,10 @@ def initialize(query_info)
       @query_info = query_info
     end
 
-    def parse(string)
+    def parse(string, query_options = {})
```

Comment:
let's remove the default here (`= {}`). It's internal use only and should always require the param, such that test don't pass by accident.